init.go

```diff
@@ -103,7 +108,7 @@ func (a *App) init() (err error) {
 				return false, err
 			}
 
-			if err := a.UpdateCache(c.Context()); err != nil {
+			if err := a.UpdateCache(c.Context(), "roles"); err != nil {
```

Comment:
There should be a comment explaining `roles`.
- What does it do?
- Why do we need it?

Reply:
Done